PublishParasoftTestResults/task.json

```diff
@@ -44,6 +44,14 @@
             "required": false,
             "helpMarkDown": "Folder to search for the test result files. Defaults to $(System.DefaultWorkingDirectory)."
         },
+        {
+            "name": "parasoftToolOrJavaRootPath",
+            "type": "string",
+            "label": "Parasoft Tool or Java Root Path",
```

Comment:
I'm not sure about the names but let's keep it for now until we get feedback from Nathan.
For the helpMarkDown, let's change it to:
Absolute location for root path of Parasoft Tool or Java installation. This path is used to optimize the conversion efficiency when processing reports. If Java is found at the specified path, it will be utilized. Otherwise, the built-in Node.js will be used as a fallback.